cfgov/regulations3k/models/django.py

```diff
@@ -292,6 +292,21 @@ def extract_graphs(self):
             'dupes': dupes,
         }
 
+    def validate_unique(self, *args, **kwargs):
+        super().validate_unique(*args, **kwargs)
+
+        # Ensure that the section's label is unique within this version.
+        qs = self.__class__.objects.filter(
+            subpart__version=self.subpart.version,
+            label=self.label
+        ).exclude(
+            pk=self.pk
+        )
+        if qs.exists():
```

Comment:
Something slightly descriptive for this variable would be nice, but I'm not sure I can come up with something more concise than `sections_with_duplicate_labels`. What do you think?

```suggestion
        sections_with_duplicate_labels = self.__class__.objects.filter(
            subpart__version=self.subpart.version,
            label=self.label
        ).exclude(
            pk=self.pk
        )
        if sections_with_duplicate_labels.exists():
```